src/raftstore/store/config.rs

```diff
@@ -17,8 +17,8 @@ use std::time::Duration;
 use raftstore::Result;
 
 const RAFT_BASE_TICK_INTERVAL: u64 = 100;
-const RAFT_HEARTBEAT_TICKS: usize = 3;
-const RAFT_ELECTION_TIMEOUT_TICKS: usize = 15;
+const RAFT_HEARTBEAT_TICKS: usize = 10;
+const RAFT_ELECTION_TIMEOUT_TICKS: usize = 50;
```

Comment:
Any estimation on the recovery time when leader is down? We should establish a constant rule for setting the heartbeat and election timeout, such as `HeartbeatTimeout = A * MaxNodeRTT + B * RecoveryTime + OtherFactors` and `ElectionTimeout = N * HeartbeatTimeout + OtherFactors`.